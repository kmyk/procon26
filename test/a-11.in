11011111111111111111111111111111
11011111111111111111111111111111
00011111111111111111111111111111
11111111111111111111111111111111
10000011111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

2
01000000
11000000
00000000
00000000
00000000
00000000
00000000
00000000

00000000
11111000
00000000
00000000
00000000
00000000
00000000
00000000
