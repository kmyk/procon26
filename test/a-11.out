
1 3 H 0
